python/src/IdentityMatrix.i

```diff
@@ -13,7 +13,7 @@ namespace OT {
 %extend IdentityMatrix {
   IdentityMatrix(const IdentityMatrix & other) { return new OT::IdentityMatrix(other); }
 
-PyObject * __setitem__(PyObject * args, Scalar val)
+static PyObject * __setitem__(PyObject * , Scalar )
```

Comment:
Why static ? Is there a warning after swig translates this ?

Reply:
Yes